### ISSUE TITLE
GraphQLError: Cannot query field "nodes" on type "RootQueryType"

### ISSUE THREAD

**Issue by DigitalGoldfish** [gentics/gatsby-source-mesh#1]
I tried to get this working with the latest version of Gatsby (1.9.175) and the latest beta of Mesh downloaded from getmesh.io. However when starting gatsby (either in build or develop) mode I get the above error message. 
Any hints on how to fix this?

**Comment by Jotschi** [comment]
Can you show me your `gatsby-config.js`? I have just bumped and tested version 1.9.175 with the [example](https://github.com/gentics/gatsby-mesh-example).

**Comment by DigitalGoldfish** [comment]
Nevermind, it had nothing to do with the gatsby-config.js. I also had a graphql query from an earlier attempt to get the mesh source plugin working in gatsby-node.js. This was the code that was throwing the above error, not the code in gatsby-config.js. Once I figured that out and removed it I could get plugin running (after a lot more cursing and debugging). 

As at first it only returned me one node. After a lot of cursing I then finally figured out that the access rights of the anonymous users were the problem (not set recursively) - Maybe worth of a FAQ entry somewhere. 

Though not 100% sure why it failed, the graphql query should have worked too, at least it does in the graphql query tool. Anyway, I will probably figure it out while building the rest of the example page.
